### ISSUE TITLE
debugfs: xorg-server debug symbols not available

### ISSUE THREAD

**Issue by stapelberg** [distr1/distri#45]
```
% ls -l /ro-dbg/debug/.build-id/f5 
total 0
lr--r--r-- 1 root root 0 2019-10-10 09:30 929a5939a91fa75dbdb512bde4827351ed577a.debug -> ../../../gtk+-amd64-3.24.0-5/debug/.build-id/f5/929a5939a91fa75dbdb512bde4827351ed577a.debug
```

but:

```
% ls -l /ro-dbg/xorg-server-amd64-1.20.3-4/debug/.build-id/f5
total 16M
-rwxr-xr-x 1 root root 16M 2019-08-09 23:19 9451c6dadbb18e6194b1e8b2745c14681494a0.debug
```

I verified that meta.binaryproto in distri/build/debug is up to date. Something seems to be wrong with the debug overlay directory.

**Comment by stapelberg** [comment]
Ah, I needed an accompanying .meta.textproto file:

```
% touch /rodebug/xorg-server-amd64-1.20.3-4.meta.textproto
```

We should make `distri fuse` cope with packages that lack a meta.textproto, or ensure they are present in the debug section, too.

**Comment by stapelberg** [comment]
Fixed with commit 7cec17f1545ec36f8cd0934a8c1cde398f7e10ef
